core/src/main/java/tc/oc/pgm/action/ActionParser.java

```diff
@@ -414,6 +397,15 @@ public FillAction parseFill(Element el, Class<?> scope) throws InvalidXMLExcepti
         parser.parseBool(el, "events").orFalse());
   }
 
+  @MethodParser("team-alias")
+  public TeamAliasAction parseTeamAliasAction(Element el, Class<?> scope)
+      throws InvalidXMLException {
+    String teamId = Node.fromRequiredAttr(el, "team-id").getValue();
```

Comment:
Should be parsing a feature reference to a team factory, that ensures that the team must exist in the map, and if it doesn't will throw corresponding xml parsing error:

```suggestion
    var team = parser.reference(TeamFactory.class, el, "team").required();
```
Also note, we do not use "team-id" anywhere in pgm, things are never suffixed "-id", so it should just be `"team"`

Also note, team could be optional and have two implementations:
 - if team is informed, the current match-scoped action runs
 - if team is not informed, a team-scoped action runs instead

As an example you can see how `SetVariableAction` parsing also has a `SetVariableAction.Indexed` variant, this is all an implementation detail, towards the user it's just as if this action (team-alias) works either with or without a specified team.